vuepress/docs/docs/README.md

```diff
@@ -44,7 +44,7 @@ Let's create a case class named `Queries` that will represent our API, with 2 fi
 
 ```scala
 // schema
-case class CharacterName(name: String)
+case class Character(name: String)
```

Comment:
`CharacterName` was not meant to be the same thing as `Character`, it was only a wrapper for the name argument. But because I simplified `Character` so much, they are identical. I suggest instead to keep `CharacterName` and add other fields in `Character`.